src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,213 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    values: {
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    },
+    errors: {
+      title: false,
+      imgUrl: false,
+      imdbUrl: false,
+      imdbId: false,
+    },
+    isNotValid: false,
+  };
+
+  handleChange = (event) => {
+    const { value, name } = event.target;
+
+    this.setState(state => ({
+      values: {
+        ...state.values,
+        [name]: value,
+      },
+      errors: {
+        ...state.errors,
+        [name]: false,
+      },
+      isNotValid: false,
```

Comment:
```suggestion
      isValid: true,
```
it's easier to read it this way